docker-compose.yml

```diff
@@ -148,3 +152,68 @@ services:
     #  - 6379:6379
     networks:
       - simulation
+
+  api:
+    image: web3labs/epirus-free-api:latest
+    environment:
+      - NODE_ENDPOINT=http://foundry:8541
+      - MONGO_CLIENT_URI=mongodb://mongodb:27017
+      - REINDEX_ENDPOINT=http://ingestion/reindex/
+      - MONGO_DB_NAME=epirus
+      - MONGO_CREATE_INDICES=true
+      - REDIS_HOST=redis
+      - REDIS_PORT=6379
+    depends_on:
+      - redis 
+      - mongodb
+      - foundry
+    networks:
+      - simulation
+  
+  mongodb:
+    image: mongo:5.0.8
+    environment:
+      - COMPOSE_HTTP_TIMEOUT=900
+      - DOCKER_CLIENT_TIMEOUT=900
+    entrypoint: mongod --bind_ip "0.0.0.0"
+    networks:
+      - simulation
+
+  web:
+    image: web3labs/epirus-free-web:latest
+    environment:
+      - API_URL=/api
+      - WS_API_URL=ws://localhost:8090
+      - DISPLAY_NETWORK_TAB=disabled
+    depends_on:
```

Comment:
we would need something like
```
ports
  - 3001:3000
```

afaik port `3000` is the default one. and `3001` is because we have another service using the external port `3000` (grafana)

Reply:
with that we shold be able to `http://localhost:3001` to access the block explorer.

Reply:
implemented